### PR TITLE
Release changes for v0.5.0

### PR DESCRIPTION
**Pull request by lucafavatella**
Depends on https://github.com/aeternity/protocol/pull/13

Finishes:
* https://www.pivotaltracker.com/story/show/154392336
* https://www.pivotaltracker.com/story/show/154393238

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-(See also [Oracles - intended usage](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/oracles/oracle_ws_api.md).
+[This release](https://github.com/aeternity/epoch/releases/tag/v0.5.0) is focused on oracles: the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_api.md) is documented and can be used to interact with Oracles on the block chain (see also [Oracles - intended usage](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/oracles/oracle_ws_api.md)).
+
+This release also:
+* Refines the HTTP API by replacing Base64 encoding with Base58Check;
+* Expands the HTTP API for enabling building apps on top of it;
+* Improves the stability of the testnet.
+
 This release introduces backward incompatible changes in the chain format.
+After upgrading your node, you will not have your previous balance (even if you keep your key pair).
```
Comment (hanssv) [inline RELEASE-NOTES.md]: Good point!

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -270,347 +264,18 @@ curl http://127.0.0.1:3003/v2/top
...
 curl http://31.13.248.102:3013/v2/top
-{"hash":"HcebFAby1g7uLgj5KEp3jBsFnKXYEthrP5+r5P2BE9Q=","height":...}
+{"hash":"bh$2UWBL9BciGC1w2FUukJZinchGRrCuwEuFTkcVvpZcfcpjiAbUy","height":...}
 ```
 
 This is the hash of the block being at the top of the chain of the node and it should be same as the hash in `prev_hash` of the block you're currently mining:
 ```
 curl http://localhost:3113/v2/block/pending
-{"data_schema":"BlockWithTxsHashes","height":... ,"prev_hash":"HcebFAby1g7uLgj5KEp3jBsFnKXYEthrP5+r5P2BE9Q=", ...}
+{...,"height":... ,"prev_hash":"bh$2UWBL9BciGC1w2FUukJZinchGRrCuwEuFTkcVvpZcfcpjiAbUy", ...}
```
Comment (lucafavatella) [inline RELEASE-NOTES.md]: @velzevur I simplified this line a bit.

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,21 +1,25 @@
 # About this release
 
-[This release](https://github.com/aeternity/epoch/releases/tag/v0.4.1) is focused on stability of the testnet - the public test network of nodes and introduces some API for apps usability build on top of it.
-In this release the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_api.md) is now documented and can be used to interact with Oracles on the block chain
-(See also [Oracles - intended usage](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/oracles/oracle_ws_api.md).
+[This release](https://github.com/aeternity/epoch/releases/tag/v0.5.0) is focused on oracles: the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_ws_api.md) is documented and can be used to interact with Oracles on the block chain (see also [Oracles - intended usage](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_api_usage.md)).
```
Comment (lucafavatella) [inline RELEASE-NOTES.md]: @hanssv I updated this section to recent https://github.com/aeternity/protocol repo

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -25,38 +29,28 @@ The binaries are tested on the following platforms:
...
-The core nodes of the public test network are accessible from the Internet and expose [an HTTP API](https://github.com/aeternity/epoch/blob/v0.4.1/config/swagger.yaml).
+The core nodes of the public test network are accessible from the Internet.
 
 Information, e.g. height, of the top block of the longest chain as seen by these core nodes of the testnet can be obtained by opening in the browser any of the following URLs:
 * http://31.13.248.103:3013/v2/top
 * http://31.13.248.102:3013/v2/top
 * http://31.13.248.105:3013/v2/top
 
-
-### Using Docker
```
Comment (lucafavatella) [inline RELEASE-NOTES.md]: @dincho I summarized this section in a line above. Also I avoided the link "docs/docker-testnet.md" as not easy to follow for the user IMO.

**Comment by dincho** [comment]
As a side note: I don't like the "this location" references. The link should be attached to it's subject.

File: RELEASE-NOTES.md (hunk 2 above)
Comment (velzevur) [reply RELEASE-NOTES.md]: Yes, this is better

**Comment by lucafavatella** [comment]
I see two approvers. CI not impacted. Merging.

**Pull request merged**